data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -2852,6 +2852,61 @@
 ]
 },{
   "type" : "recipe",
+  "result": "compost_unf",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_CHEMICALS",
+  "skill_used": "survival",
+  "skills_required": ["survival", 0],
+  "difficulty": 1,
+  "time": 360000,
+  "reversible": false,
+  "autolearn": true,
+  "qualities":[
+    {"id":"DIG","level":2,"amount":1}
+	          ],
+  "components": [
+    [ [ "water", 1 ], [ "water_clean", 1 ] ,
+	[ "apple",        2 ], 
+    [ "apricot",      2 ], 
+    [ "banana",       2 ], 
+    [ "blackberries", 2 ], 
+    [ "blueberries",  2 ], 
+    [ "cherries",     2 ], 
+    [ "cranberries",  6 ], 
+    [ "grapes",       4 ], 
+    [ "kiwi",         2 ], 
+    [ "lemon",        2 ], 
+    [ "mango",        2 ], 
+	["mushroom", 4 ],
+	[ "mushroom_morel", 4 ],
+    [ "orange",       2 ], 
+    [ "papaya",       2 ], 
+    [ "peach",        2 ], 
+    [ "pear",         2 ], 
+    [ "pineapple",    2 ], 
+    [ "plums",        2 ], 
+    [ "pomegranate",  2 ], 
+    [ "raspberries",  2 ], 
+    [ "strawberries", 2 ], 
+    [ "rhubarb",      2 ], 
+    [ "coconut",      2 ], 
+	[ "broccoli",   2 ], 
+    [ "tomato",     2 ], 
+    [ "pumpkin",    2 ], 
+    [ "zucchini",   2 ], 
+    [ "celery",     2 ], 
+    [ "potato_raw", 2 ], 
+    [ "onion",      2 ], 
+    [ "carrot",    12 ], 
+    [ "cabbage",    2 ], 
+    [ "lettuce",    8 ], 
+    [ "veggy",      2 ],
+    [ "veggy_wild", 2 ],
+    [ "dandelion_cooked", 2 ],
+    [ "wild_herbs", 40 ]
+  ]]
```

Comment:
Being able to craft fertilizer from common food items leads to a situation where you can produce more food if you sacrifice some of the produce to make fertilizer.